src/librbd/cache/ParentCacheObjectDispatch.cc

```diff
@@ -1,14 +1,18 @@
 // -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
 // vim: ts=8 sw=2 smarttab
 
+#include "common/errno.h"
 #include "librbd/ImageCtx.h"
 #include "librbd/Utils.h"
 #include "librbd/asio/ContextWQ.h"
+#include "librbd/io/AioCompletion.h"
```

Comment:
I think you might hit dynamic linker issues now unless you move these to a static library that is also linked against this plugin.